packages/core/src/serializer/elements.ts

```diff
@@ -40,22 +41,65 @@ import { InvalidStaticFile } from '../workspace/static_files/common'
 // The deserialization process recover the information by creating the classes based
 // on the _salto_class field, and then replacing the placeholders using the regular merge method.
 
+// Do not use the class's name for serialization since it can change (e.g. by webpack)
+/* eslint-disable object-shorthand */
+const NameToType = {
+  InstanceElement: InstanceElement,
+  ObjectType: ObjectType,
+  Variable: Variable,
+  PrimitiveType: PrimitiveType,
+  ListType: ListType,
+  Field: Field,
+  TemplateExpression: TemplateExpression,
+  ReferenceExpression: ReferenceExpression,
+  VariableExpression: VariableExpression,
+  StaticFile: StaticFile,
+}
+
+type SerializableTypeNames = keyof typeof NameToType
+type SerializableTypes = InstanceType<types.ValueOf<typeof NameToType>>
```

Comment:
do you think it would be better if the type name is singular and not plural?
```suggestion
type SerializableTypeName = keyof typeof NameToType
type SerializableType = InstanceType<types.ValueOf<typeof NameToType>>
```

Reply:
yes I think you are right

Reply:
Actually I renamed to `SerializedName` and `Serializable`, WDYT? If you don't like `Serializable` I can do `SaltoSerializable` or something